go.mod

```diff
@@ -86,6 +86,7 @@ require (
 	golang.org/x/sync v0.7.0 // indirect
 	golang.org/x/term v0.22.0 // indirect
 	golang.org/x/text v0.16.0 // indirect
+	gopkg.in/go-jose/go-jose.v2 v2.6.3 // indirect
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
```

Comment:
Two diff module paths `/square` vs `/go-jose` does that even fix the issue?

Reply:
FYI I pulled the branch and looked at the deptree from this project and the only go-jose tree I see is from zitadel so if 2.6.3 has the vulnerability fixed, it seems to me that this would resolve the issue here